stylesheets/style-about.css

```
@@ -0,0 +1,175 @@
+/* 242F9B  646FD4  9BA3EB   DBDFFD */
+
+img {
+    height: 40vh;
+    object-fit: cover;
+    width: 40vw;
+} 
+
+h1 {
+    padding-bottom: 10px;
+}
+
+h3 {
+    font-size: 1.5em;
+    padding-left: 20px;
+    line-height:0%;
+}
+
+#container-about {
+    display: grid;
+    grid-template-rows: .2fr .75fr .15fr .3fr .15fr .75fr .1fr;
+}
+#intro {
+    background-color: #242F9B;
+    color: #DBDFFD;
+    /* background-color: #9BA3EB ; */
+    grid-row: 2/3;
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    column-gap: 0em;
+    grid-template-rows: 1fr .25fr .75fr;
+    margin-top: 50px;
+}
+
+#intro img {
+    margin-left: 4em;
+    grid-column: 1/2;
+    grid-row: 1/2;
+    align-self:end;
+}
+
+#intro p {
+    color:black;
+    padding: 15px;
+    background-color: #9BA3EB ;
+    grid-column: 2/3;
+    grid-row: 1/2;
+    max-width: 25em;
+    line-height: 1.3em;
+    font-size: larger;
+    justify-self: center;
+    align-self:end;
+}
+
+#intro h3 {
+    margin-left: 3em;
+    background-color: #9BA3EB ;
+    grid-column: 1/3;
+    grid-row: 2/3;
+    display:flex;
+    align-self:flex-end;
+} 
+
+#intro ul {
+    margin-top: 0;
+    grid-column: 1/3;
+    grid-row: 3/4;
+    line-height: 1.3em;
+    font-size: larger;
+    display: flex;
+    flex-direction: column;
+}
+#intro ul {
+    max-width: 40em;
+}
+#intro ul li {
+    margin-left: 4em;
+    margin-bottom: 5px;
+    list-style-type:circle;
+}
+#title-work {
+    grid-row: 3/4;
+    padding: 30px 5px;
+    /* 646FD4  9BA3EB  */
+    background-color: #DBDFFD;
+    /* background-color: #242F9B; */
+    color:#242F9B;
+    /* color: #DBDFFD; */
+}
+#section-work {
+    grid-row: 4/5;
+}
+#section-work ul{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap: 0px 0px;
+}
+#resume {
+    grid-column:1/2;
+    align-self: center;
+    font-size:xx-large;
+    display: flex;
+    justify-self: center;
+}
+figure {
+    font-size:4em;
+    padding:0;
+    margin:0;
+}
+
+#OT {
+    grid-column:2/3;
+    align-self: center;
+    display: flex;
+    justify-self: center;
+}
+
+#OT img {
+    height: 25vh;
+    width: 15vw;
+    display: flex;
+    justify-self: center;
+}
+#AmeriCorps {
+    grid-column:3/4;
+    align-self: center;
+    display: flex;
+    justify-self: center;
+}
+#AmeriCorps img {
+    height: 25vh;
+    width: 15vw;
+    display: flex;
+    justify-self: center;
+}
+#Habitat{
+    grid-column:4/5;
+    align-self: center;
+}
+
+#Habitat img {
+    height: 25vh;
+    width: 15vw;
+}
+
+
+#title-family {
+    grid-row: 5/6;
+    padding: 30px 5px;
+    background-color: #242F9B;
+    color: #DBDFFD;
+}
+#section-hobbies {
+    grid-row: 6/7;
+    border: solid blue 2px;
+}
```

Style: This should be named without the tag in the name, that way you don't need to change the id if you change it to an aside or something:

```suggestion
#hobbies-wrapper {
    grid-row: 6/7;
    border: solid blue 2px;
}
```